pkg/target/target_test.go

```diff
@@ -494,6 +497,7 @@ func TestToMatcher(t *testing.T) {
 			constraint: fooConstraint(),
 			want: &Matcher{
 				fooMatch(),
```

Comment:
Use named fields for clarity instead of the implicit field ordering. So

```
want: &Matcher{
  match: fooMatch(),
  cache: newNsCache(nil),
}
```